src/help.rs

```diff
@@ -89,6 +89,7 @@ pub fn help() {
     printtr!("getpkgbuild specific options:");
     printtr!("    -p --print            Print pkgbuild to stdout");
     printtr!("    -c --comments         Print AUR comments for pkgbuild");
+    printtr!("    -e --ssh              Fetch using a read-write remote (e.g.: via ssh)");
```

Comment:
just "Clone over ssh" or something would be sufficient. There's no need to abstract away that ssh is used under the hood. If you want the detail that it can be written to then that can be in the man page.